apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -198,7 +207,7 @@ class PageTemplateModal extends Component {
 					<Button
 						isPrimary
 						isLarge
-						disabled={ isEmpty( previewedTemplate ) || isLoading }
+						disabled={ isEmpty( previewedTemplate ) || isLoading || wasClosed }
```

Comment:
I still don't understand why this button is disabled if we've previously closed the model. Surely it won't be visible. I'm probably missing something...

Reply:
When I was testing, getting back to the previous took a kinda long time, allowing me to add a template in the editor even after closing the modal. A few seconds after adding the template into the editor, the browser finally went to the previous page.
Although I couldn't reproduce it again (no gif :-() I think it's possible so I suggest disable the button in order to avoid this situation.

Reply:
Ah I see. Thanks. That's an interesting edge case.